gcp_variant_transforms/bq_to_vcf.py

```diff
@@ -228,7 +255,27 @@ def _get_file_path_and_sorted_variants((file_name, variants), file_path_prefix):
   yield (file_path, sorted(variants))
 
 
-def _pair_variant_with_key(variant, number_of_variants_per_shard):
+def _is_direct_runner(pipeline):
+  return isinstance(pipeline.runner, direct_runner.DirectRunner)
+
+
+def _write_source_file_to_target_file(source_file, target_file):
+  with filesystems.FileSystems.open(source_file) as f:
+    for line in f:
+      target_file.write(line)
+
+
+def _pair_variant_with_key(variant,
+                           is_direct_runner,
+                           number_of_variants_per_shard):
+  # type: (vcfio.Variant, bool, int) -> Tuple[str, vcfio.Variant]
+  """Pairs the variant with a key.
+
+  For direct runner, all variants are paired with the same key. Otherwise, the
+  key is determined by the variant's reference name and start position.
+  """
+  if is_direct_runner:
+    return (_LOCAL_TEMP_DATA_FILE_NAME, variant)
   return ('%s_%011d' % (variant.reference_name,
```

Comment:
I just realized that we should do `from __future__ import division` at the top so that we get python3 division behavior. You should then do `//` here to get an integer. See https://www.python.org/dev/peps/pep-0238/ for details.